### DISCUSSION TITLE
What can the "backend integration" be used for?

### DISCUSSION THREAD

**Discussion by pdolif** [rwth-acis/Social-Bot-Framework#80] [Q&A] [answered]
In the wiki I've read about the backend integration.
It says:

> For the bot to be triggered, the service must send an appropriate [monitoring message](https://github.com/rwth-acis/mobsos-data-processing/wiki/Manual#2-monitor-a-service).

```
{
    "serviceAlias": "",
    "functionName": "",
    "attributes":{}
}
```
Which functions can be triggered in this way?
I would like to use a las2peer service to trigger the bot to send a message to a specific RocketChat channel. Would that be possible?

**Comment by Aran30** [comment, marked as answer]
To be honest, I am not even sure if the intended original functionality still works, ergo you can't trigger bot functions with those monitoring messages (might be wrong though).

Sending/triggering specific rocket chat bot messages to a user using a rest call is not implemented as far as I'm aware.

**Reply by pdolif** [reply]
Thanks for the quick reply!
